load_average/load_average.tf

```diff
@@ -0,0 +1,71 @@
+resource "datadog_monitor" "load_average_1" {
+  count   = "${var.monitor_enabled}"
+  name    = "High load average 1 ${module.label.id}"
+  type    = "${var.alert_type}"
+  message = "High load average 1 last ${var.load_average["period"]} on host: {host.name} with IP: {host.ip} ${var.notify}"
+  query   = "avg(last_${var.load_average["period"]}):avg:system.load.1{${join(",", compact(var.datadog_monitor_selector))}} by {host} > ${var.load_average["critical_threshold"]}"
```

Comment:
Don't see why we need the `datadog_monitor` prefix. Everything is related to `datadog_monitor` by nature of the resource type.